lib/popup.js

```diff
@@ -254,6 +254,6 @@ function getExtension(lang) {
     case 'C++':
       return '.cpp'
     default:
-      return '.${lang.toLowerCase()}';
+      return '.' + lang.toLowerCase();
```

Comment:
Thanks for fixing this, given we have this we can remove Java, Go, Scala and Swift is it?

Reply:
That's a good idea. I'll rework the code.

Reply:
Reworked. I also added support for the rest of the supported Leetcode languages and sorted the case matches by name.

Reply:
Thanks

Reply:
You're welcome